binder/src/androidTest/java/io/grpc/binder/internal/BinderClientTransportTest.java

```diff
@@ -188,6 +200,9 @@ public void testRequestWhileStreamIsWaitingOnCall_b154088869() throws Exception
 
   @Test
   public void testTransactionForDiscardedCall_b155244043() throws Exception {
+    transport = new BinderClientTransportBuilder().build();
+    transport.start(transportListener).run();
```

Comment:
Consider putting these two lines into a new startTransport() method to mirror AbstractTransportTest. (Your call though).

Reply:
"These two" -> start() + awaitReady().

Reply:
done